java/src/main/java/org/rocksdb/ExportImportFilesMetaData.java

```diff
@@ -0,0 +1,56 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * The metadata that describes a column family.
+ */
+public class ExportImportFilesMetaData {
+  private final byte[] dbComparatorName;
+  private final LiveFileMetaData[] files;
+
+  /**
+   * Called from JNI C++
+   */
+  public ExportImportFilesMetaData(final byte[] dbComparatorName, final LiveFileMetaData[] files) {
+    this.dbComparatorName = dbComparatorName;
+    this.files = files;
+  }
+
+  /**
+   * The name of the db comparator.
+   *
+   * @return the dbComparatorName
+   */
+  public byte[] dbComparatorName() {
+    return dbComparatorName;
+  }
+
+  /**
+   * The metadata of all files in this column family.
+   *
+   * @return the levels files
+   */
+  public List<LiveFileMetaData> files() {
+    return Arrays.asList(files);
+  }
+
+  public long newExportImportFilesMetaDataHandle() {
```

Comment:
I don't think this should be in this class. I see it is called from RocksDB. I think it would be better if the processing was done in RocksDB as the native handles are not used here.